### ISSUE TITLE
Rand seed doesn't seem to work in 3.4+ only in 2.7.x

### ISSUE THREAD

**Issue by ndanielsen** [joke2k/faker#367]
I can't get the random seed to work in 3.4+ only in 2.7 with fake-factory==0.5.9

`from faker import Factory, Generator, Faker

fake = Factory.create()
fake.seed(4321)

for _ in range(10):
    print(fake.name())
`
Am I missing something?

**Comment by jeffwidman** [comment]
Possible dupe of #363

**Comment by adamchainz** [comment]
Most likely is a dupe. Try a random generator that's not backed by a dictionary (you'll have to read the source to find one).

**Comment by IlyaSemenov** [comment]
It works for me in 0.6.0 on Python 3.5. Always returning the same names.

**Comment by felixhummel** [comment]
Just for future reference (workaround until #363 is fixed):
```
$ python -V
Python 3.5.1

$ cat foo.py 
from faker import Faker
fake = Faker()
fake.random.seed(4321)
print(fake.name())

$ PYTHONHASHSEED=23 python foo.py 
Rhonda Peterson
$ PYTHONHASHSEED=23 python foo.py 
Rhonda Peterson
```

See also https://docs.python.org/3.3/using/cmdline.html#cmdoption-R for what changed since 3.3.

**Comment by adamchainz** [comment]
Try now after #384 ?

**Comment by ananthkm1** [comment]
The below code is not working:

from faker import Faker
fk = Faker()
print(fk.name)

error:
faker has no name member.

using anaconda1.8.7 in windows8. installed faker using "conda install faker" in dedicated envi. I am unable see package in conda navigator under installed list. Unable to install packages through navigator.

**Comment by fcurella** [comment]
@ananthkm1 Could you open a separate issue for anaconda?

**Comment by fcurella** [comment]
Closing this for inactivity
